flight/PiOS/STM32/pios_servo.c

```diff
@@ -460,6 +503,13 @@ void PIOS_Servo_SetFraction(uint8_t servo, uint16_t fraction,
 			 */
 			output_channels[servo].i.dshot.value = fraction >> 5;
 			return;
+		case SYNC_DSHOT_DMA:
+#if defined(PIOS_INCLUDE_DMASHOT)
+			if (PIOS_DMAShot_IsConfigured()) {
```

Comment:
If we wrap DMA_Cmd in a macro and place an #ifdef in _DMASetup(), it can probably work. For F0 and also F3 I guess.